contracts/eos/BancorNetwork/BancorNetwork.hpp

```diff
@@ -25,35 +25,85 @@ using namespace std;
  * - For example, in order to convert 10 EOS into BNT, the caller needs to transfer 10 EOS to the contract
  * and provide the following memo:
  * > `1,bnt2eoscnvrt BNT,1.0000000000,receiver_account_name`
+ * - Optionally, an affiliate fee and affiliate account may be included (either both or neither) as such:
+ * > `1,PATH,1.0000000000,receiver_account_name,affiliate_account_name,affiliate_fee`
  * @{
 */
 
+/**
+ * @details 
+ * - the account which originated the entire conversion path
+ * - the final destination of the entire conversion path
+ * - the contract where this conversion was processed
+ * - the return from the conversion in BNT before fee deduction
+ * - the account that was paid the affiliate fee
+ * - the amount of that was paid as affiliate fee 
+ */
+#define EMIT_AFFILIATE_EVENT(sender, destination, from_contract, fee_account, to_amount, fee_amount){ \
```

Comment:
EMIT_AFFILIATE_EVENT -> EMIT_AFFILIATE_FEE_EVENT

Also, I think the affiliate account/amount should be enough.
Thoughts?

Reply:
Can remove destination and return but at least need from_contract, portals might want to have a websocket listening for payments on specifically their multiConverter

Reply:
So remove them, destination and return?

Reply:
Done